### ISSUE TITLE
How to judge the convergence of the pre-training model？

### ISSUE THREAD

**Issue by PamelaDDD** [ChenRocks/UNITER#89]
How to measure the loss weight of different pre-training tasks? Which task's loss determines the model training convergence?
